microservices/notification-service/src/scrim/scrim.service.ts

```diff
@@ -1,4 +1,5 @@
 import {Injectable, Logger} from "@nestjs/common";
+import {MICROSERVICES_PACKAGE_NOT_FOUND_EXCEPTION} from "@nestjs/core/errors/messages";
```

Comment:
What does this import do? It doesn't appear to be used.

Reply:
It does nothing. It was an autoimport from a bad autocomplete of microserviceOptions . 
making the changes and will push them shortly